dkist/io/asdf/converters/file_manager.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+from asdf.extension import Converter
+
+
+class FileManagerConverter(Converter):
+    tags = [
+        "tag:dkist.nso.edu:dkist/array_container-0.2.0",
+        "asdf://dkist.nso.edu/array_container-1.0.0",
+    ]
+    types = ["dkist.io.file_manager.FileManager"]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from dkist.io.file_manager import FileManager
+        from dkist.io.loaders import AstropyFITSLoader
+
+        filepath = Path((ctx.url or ".").replace("file:", ""))
```

Comment:
Maybe raise an error here if the URL has some other scheme besides `file:` ?  Just so the user gets a more specific message and not a failure down the line from trying to open an http URL as a file path.

Reply:
In *theory* I would want the `FileManager` class to handle loading over http, so it should be ok if this returns something other than a file URL.

Reply:
In the end due to a *super* annoying windows bug I ended up erroring!